qdb/tracer.py

```diff
@@ -123,6 +153,25 @@ def _init(self, config=None, merge=False, **kwargs):
                 RemoteOutput(self.cmd_manager, '<stderr>'),
             )
 
+    def bound_cmd_manager():
+        def fget(self):
+            return self.__cmd_manager
+
+        class BoundCmdMangaer(object):
```

Comment:
> and I will move the class up to mod scope

Can you stick a docstring on it explaining that it's a proxy object that pretends to be the CommandManager instance but forwards the tracer to the first argument to all the CommandManager's methods?